app/build.gradle

```diff
@@ -1,126 +1,126 @@
 plugins {
-    id 'com.android.application'
```

Comment:
please keep one tab space here and don't change anywhere tab space. In issue i have mentioned to remove unnecessary spaces from files. eg :  1<lines spaces.